ports/nrf/boards/pca10056/mpconfigboard.h

```diff
@@ -82,3 +82,6 @@
 #endif
 
 #define HELP_TEXT_BOARD_LED         "1,2,3,4"
+
+#define PORT_HEAP_SIZE                (64*1024)
```

Comment:
Should this be larger? The linker script has `_minimum_heap_size = 128K;`, and there is plenty of free SRAM on this SoC for the heap.

Reply:
How much do you recommend?